### ISSUE TITLE
Update to RxDart 0.21.0

### ISSUE THREAD

**Issue by shinayser** [lejard-h/flutter_google_places#53]
The current version of the library is uncompilant with the latest version of RxDart (0.21.0):

![image](https://user-images.githubusercontent.com/1146982/54762618-3328b800-4bd3-11e9-884b-f61bd79406b6.png)

**Comment by athornz** [comment]
While waiting for my PR to be merged and published, you can use my branch:

```
  flutter_google_places:
    git: git@github.com:athornz/flutter_google_places.git
```

**Comment by lejard-h** [comment]
fix on `0.2.2`

**Comment by rdev-software** [comment]
Can you boost it to 0.22.0 ?
